app/models/defendant.rb

```diff
@@ -3,7 +3,7 @@
 class Defendant
   include ActiveModel::Model
 
-  attr_accessor :body
+  attr_accessor :body, :gender, :address_1, :address_2, :address_3, :address_4, :address_5, :postcode
```

Comment:
I think we are missing the initilization here (apart from the `body`)

Reply:
`:gender, :address_1, :address_2, :address_3, :address_4, :address_5, :postcode` aren't actually part of the defendant data we get from the prosecution_case search results, so they won't appear in the body. That's the data we need to get by doing subsequent calls to get additional hearing data. 

My original thought was that we could update the `body` with that data after doing those calls, but after thinking about it it seemed easier to hold them as separate attributes on the model, hence this change.

You're right that they don't get initialised here, they would only get set later on, after we've done those extra hearing calls.

Reply:
makes sense. I'm happy to have them in as it complements your serializer specs.